src/util/kafka/StreamHandler.cpp

```diff
@@ -0,0 +1,93 @@
+#include <string>
```

Comment:
Lets add a standard license header to this file

Reply:
Done.

Reply:
Ok